stacker/hooks/aws_lambda.py

```diff
@@ -116,22 +118,36 @@ def _find_files(root, includes, excludes):
     """
 
     root = os.path.abspath(root)
-    file_set = formic.FileSet(directory=root, include=includes,
-                              exclude=excludes)
+    if walk is True:
```

Comment:
should just be `if walk:`

Reply:
You can actually also avoid this if/else and the `_walk` function entirely with `funtools.partial` like this:

```
file_set = formic.FileSet(directory=root, include=includes,
                          exclude=excludes,
                          walk=partial(os.walk, followlinks=walk)

```